src/PageContent/Calendar/EventCard.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import './EventCard.css';
+import green_truck from '../../icons/green_truck.svg';
+import grey_truck from '../../icons/grey_truck.svg';
+
+class EventCard extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            eventType: 'recurring',
+            eventTime: '10am - 12pm',
+            eventStatus: 'future'
+        };
+    }
+    render() {
+        let type = '';
+        if (this.state.eventType === 'recurring') {
+            type = 'Reccuring Pick Up';
+        }
+        return (
+            <div>
```

Comment:
instead of having 2 blocks of the same code, you could do something similar to what you're doing for `type` above? It looks like the only difference between the two cases are the `className` and which truck, so you could do something like:
```
var containerClassName = '';
var truckColor = '';
if (this.props.isPast) {
    containerClassName = 'event-container-past';
    truckColor = 'grey';
} else {
    ...
}
```
and just have 1 block that uses `{containerClassName}` and `{truckColor}`. Something along this line.